.github/workflows/nightly.yml

```diff
@@ -22,6 +22,44 @@ on:
     - cron:  '23 3 * * *'
 
 jobs:
+  sanitizer:
+    name: "Sanitizers"
+    strategy:
+      fail-fast: false
+
+      matrix:
+        include:
+          - target: sanitizer
+            compiler: msvc
+            host_os: windows-2022
+            make_tool: ninja
+          - target: sanitizer
+            compiler: gcc
+            host_os: ubuntu-24.04
+
+    runs-on: ${{ matrix.host_os }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: ./source
+
+      - name: Fetch BoringSSL fork for BoGo tests
+        uses: actions/checkout@v4
+        with:
+          repository: randombit/boringssl
+          ref: rene/runner-20240524
```

Comment:
Would be good if we could centralize this somehow. I'll have to update boring soon, anyway. For the new ML-KEM-hybrids. I can take care of this then. Let's try to collectively remember. 😅